package.json

```diff
@@ -12,7 +12,7 @@
     "test": "run-s eslint",
     "eslint": "eslint --ignore-pattern dist .",
     "jest": "cross-env PORT=8080 BABEL_ENV=jest jest",
-    "dev-server": "cross-env PORT=8080 webpack-dev-server",
+    "dev-server": "cross-env PORT=8080 webpack-dev-server --mode=development",
```

Comment:
This made it easier for me to work with the code while adding.

Reply:
Ah, nice. Thanks.